amivapi/utils.py

```diff
@@ -74,7 +74,7 @@ def get_id(item):
         return ObjectId(item['_id'])
 
 
-def mail(to, subject, text):
+def mail(to, subject, text, replyTo = None):
```

Comment:
I suggest to rename the argument to `reply_to` to conform with PEP8 code style.

You should also describe the additional argument in the documentation in the docstring above.